src/smart_contracts/ticket_handle.ml

```diff
@@ -0,0 +1,46 @@
+open Crypto
+
+module type S = sig
+  include Conversions.S
+
+  type t [@@deriving yojson, ord, eq]
+
+  val to_string : t -> string
+
+  val of_string : string -> t option
+
+  val make : Address.t -> Ticket_id.t -> Amount.t -> t
+
+  val to_bytes : t -> bytes
+
+  val of_bytes : bytes -> t option
+
+  module Table : sig
+    include Hashtbl.S with type key := t
+  end
+end
+
+module Make (CC : Conversions.S) = struct
+  include CC
+  include BLAKE2B_20
+
+  let make owner ticket amount =
+    BLAKE2B_20.hash_v
+      [
+        Address.to_string owner;
+        Tezos.Ticket_id.to_string ticket;
```

Comment:
I think that's better using `Core_deku.Ticket_id` instead of Tezos' directly.

Reply:
Core_deku.Ticket_id is just an alias to Tezos.Ticket_id.t that doesn't expose the to_string method. It will be changed in ticket_minting pr when core_deku ticket becomes more self sufficient and not barely an alias